poppy/dms.py

```diff
@@ -333,7 +333,7 @@ def get_opd(self, wave):
                     shift_y_pix = int(np.round(self.shift_y /pixscale_m))
                     interpolated_surface = np.roll(interpolated_surface, shift_y_pix, axis=0)
 
-        return interpolated_surface
+        return 2*interpolated_surface # note *2 to account for DM being reflective surface
```

Comment:
Hi @remorgan123 - I would like to _not_ include this factor of 2 here, or at least to make it optional via some class parameter. The implicit convention thus far in poppy has been to have the DM classes modeled in terms of OPD rather than physical surface (despite the variable names...).  I realize this is potentially confusing and hasn't been super well documented though.  More significantly, changing this behavior now would be introduce unexpected factor of 2 change into any existing code that uses the poppy DM classes; i.e. this is not a back-compatible change. 

Instead I think we could have this as a new parameter to the class init, e.g. `include_factor_of_two=False` or similar. And then here would have more like: 
```
coefficient = 2 if self.include_factor_of_two else 1
return coeff*interpolated_surface
```

Doing it that way would let us gradually shift the behavior here, without immediately introducing unexpected factor of 2 change for all existing uses of this code...